apps/web/src/components/features/properties/PropertyDetail.tsx

```
@@ -0,0 +1,298 @@
+              stays, this rental property accepts cryptocurrency payments for a seamless booking
+              experience.
+            </p>
+          </div>
+
+          <div className="mb-8">
+            <h2 className="text-2xl font-semibold mb-4">Amenities</h2>
+            <ul className="grid grid-cols-1 md:grid-cols-2 gap-2">
+              {property.amenities.map((amenity, index) => (
+                <li key={index} className="flex items-center">
+                  <span className="w-2 h-2 bg-blue-600 dark:bg-blue-400 rounded-full mr-2"></span>
+                  {amenity}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)